sync/cache/librariesioGoGo.json

```diff
@@ -2765,6 +2765,277 @@
     "sirupsen/logrus/examples/basic",
     "sirupsen/logrus/hooks/bugsnag",
     "sirupsen/logrus/examples/hook",
-    "sirupsen/logrus/hooks/papertrail"
+    "sirupsen/logrus/hooks/papertrail",
```

Comment:
you sure we should keep cache under version control?

Reply:
not sure git a right place, but it'd be good to make it available to anyone who's running the tool. What a better place would be?

Reply:
let's keep it then.